src/Documents/ListRecordsDocument.php

```diff
@@ -32,4 +33,20 @@ public function processOaiRecords(DataList $oaiRecords): void
         }
     }
 
+    public function setResumptionToken(string $resumptionToken): void
+    {
+        $listRecordsElement = $this->findOrCreateElement('ListRecords');
+        $resumptionTokenElement = $this->findOrCreateElement('resumptionToken', $listRecordsElement);
+
+        $resumptionTokenElement->nodeValue = $resumptionToken;
+
```

Comment:
```suggestion
        if (!$resumptionToken) {
            return;
        }
```
Can we please add this for the empty resumption token scenario? 👀
I just realised it would cause an error in the decode process(`$decode = base64_decode($resumptionToken, true);`) if we set an empty resumption token in the last page of the response. Looks `base64_decode` are not able to decode an empty string.


![Screen Shot 2022-04-04 at 2 30 40 PM](https://user-images.githubusercontent.com/81267819/161464304-f8214666-7c22-4834-90e9-aea8b740d570.png)

Reply:
Thanks, @MelissaWu-SS!

Fixed at the source of the issue:
https://github.com/silverstripe-terraformers/open-archive-initiative/pull/11/commits/fad9151d25d08bdccce8d341e3c8172e231956ca